admin/templates/spam/user.html

```diff
@@ -0,0 +1,85 @@
+{% extends "base.html" %}
+{% load render_bundle from webpack_loader %}
+{% load spam_extras %}
+{% load user_extras %}
+
+{% load static %}
+{% block title %}
+    <title>User Spam</title>
+{% endblock title %}
+{% block content %}
+    <h2>User's List of Spam</h2>
+    <a href="{{ user_id | reverse_user }}" class="btn btn-primary">
+        <i class="fa fa-arrow-left"></i>
+        User details
+    </a>
+    <div class="btn-group">
+    {% if status == '1' %}
```

Comment:
Can you pass the constants to the template so that you can do `if status == FLAGGED`, etc.?